external-crates/move/crates/move-compiler/tests/sui_mode/linter/true_unnecessary_public_entry.snap

```diff
@@ -9,18 +9,16 @@ warning[Lint W99010]: unnecessary `entry` on a `public` function
   ┌─ tests/sui_mode/linter/true_unnecessary_public_entry.move:2:12
   │
 2 │     public entry fun true_unnecessary_public_entry() {}
-  │            ^^^^^ `entry` on `public` functions limits composability, as it forces the return value to have `drop` ability. It is only meaningful in niche scenarios.
+  │            ^^^^^ `entry` on `public` functions limits composability as it adds additional restrictions, e.g. the type of each return value must have `drop`. `entry` on `public` is only meaningful in niche scenarios.
```

Comment:
```suggestion
  │            ^^^^^ `entry` on `public` functions limits composability as it adds restrictions, e.g. the type of each return value must have `drop`. `entry` on `public` is only meaningful in niche scenarios.
```
Rereading this, "additional" seems a bit silly since we say "adds restrictions", they are by definition additional